### DISCUSSION TITLE
Why does Valtio - when used in vanilla js - import React?

### DISCUSSION THREAD

**Discussion by DynamicArray** [pmndrs/valtio#172] [Q&A]
Love this library, but why does Valtio - when used in vanilla js - import React?

![image](https://user-images.githubusercontent.com/794239/125113166-3156cb80-e0e0-11eb-8c58-9d717fb7e31d.png)

**Comment by drcmda** [comment]
could it be something else in your project? react is not part of the valtio vanilla bundle https://unpkg.com/valtio@1.0.7/vanilla.js
